### PR TITLE
Add Vanilla HTML/JS feedback example

### PR DESCRIPTION
**Pull request by Munter**
This example could be used to link to from the README

### CODE REVIEW

File: HTTP-API.md [hunk 1]
```diff
@@ -105,3 +105,17 @@ POST https://tracking.bucket.co/trCqx4DGo1lk3Lcct5NHLjWy/event
...
+
+You can submit qualitative feedback related to a specific feature in order to pair your quantitative metrics with qualitative insigths. You can collect either a 1-5 satisfaction score or a comment or both.
+
+At least one of the optional fields `score` or `comment` must be submitted
+
+| field      | required |         Type |
+| ---------- | :------: | -----------: |
+| featureId  | Required |       String |
+| userId     | Required |       String |
+| companyId  | Optional |       Object |
```
Comment (roncohen) [inline HTTP-API.md:119]: `companyId` is a string

File: README.md [hunk 2]
```diff
@@ -34,6 +36,15 @@ bucket.company("acme_inc", { name: "Acme Inc", plan: "pro" }, "john_doe");
 
 // track events
 bucket.track("sent_message", { foo: "bar" }, "john_doe");
+
+// collect qualitative feedback
+bucket.feedback({
+  featureId: "my_feature_id",
+  userId: "john_doe",
+  companyId: "Acme Inc", // String (optional)
```
Comment (roncohen) [inline README.md:44]: let's keep `userId` and `companyId` in the same style. So: `acme_inc` for example

File: example/feedback/feedback.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(feedbackPayload),
+        });
+      }
+    </script>
+
+    <form action="#" onsubmit="handleSubmit(event)">
```
Comment (roncohen) [inline example/feedback/feedback.html:49]: do you need `(event)` here?

File: example/feedback/feedback.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+          userId: "EXAMPLE_USER",
+          companyId: "EXAMPLE_COMPANY",
+          score: formData.score ? Number(formData.score) : null,
+          comment: formData.comment ? formData.comment : null,
+        };
+
+        // Using the Bucket SDK
+        bucket.feedback(feedbackPayload);
+
+        // Using the Bucket API
```
Comment (roncohen) [inline example/feedback/feedback.html:38]: People have a tendency to blindly copy/paste things, so i suggest we comment out the full fetch example:

```
// Or using the Bucket HTTP API
/*
     fetch(....
     
     
*/

**Review by roncohen — APPROVED** [review]
nice @Munter! I'd love for us to also have a minimal React example at some point, but this is great start.

File: example/feedback/feedback.html (hunk 3 above)
Comment (Munter) [reply example/feedback/feedback.html:49]: Yeah. Old style inline event handling. Saves us from the additional indirection of having to attach an event handler with a querySelector in the JS. I'd like to keep the example as focused on the relevant parts as possible

File: example/feedback/feedback.html (hunk 4 above)
Comment (Munter) [reply example/feedback/feedback.html:38]: Good point. will comment out

File: README.md (hunk 2 above)
Comment (Munter) [reply README.md]: 👍

File: HTTP-API.md (hunk 1 above)
Comment (Munter) [reply HTTP-API.md]: Good catch

**Comment by Munter** [comment]
Should we merge this one asap, or hold it back to be timed with official communication about the feature?

**Comment by roncohen** [comment]
Let's wait with merging this documentation until the rest of the pieces are ready 👍

**Pull request merged**